tasks/sync_teams.py

```diff
@@ -24,7 +24,9 @@ async def run_async(self, db_session, ownerid, *, username=None, **kwargs):
         assert owner, "Owner not found"
         service = owner.service
 
-        git = get_owner_provider_service(owner, using_integration=False)
+        git = get_owner_provider_service(
+            owner, using_integration=False, ignore_installation=True
```

Comment:
Just checking if this `ignore_installation=True` meant to be `ignore_installation=(not using_integration)`?

Reply:
yes, it's supposed to be the opposite in this case.